src/pages/public_registration/PublicRegistrationAuthors.tsx

```diff
@@ -43,7 +43,8 @@ const PublicRegistrationAuthors: FC<PublicRegistrationAuthorsProps> = ({ contrib
             email,
           } = contributor;
           const affiliationIndexes = contributor.affiliations
-            ?.map((affiliation) => distinctUnits.indexOf(affiliation.id) + 1)
+            ?.map((affiliation) => affiliation.id && distinctUnits.indexOf(affiliation.id) + 1)
+            .filter((affiliationIndex) => affiliationIndex)
```

Comment:
Hadde litt problem med filter her også, jeg mener dette skal være lov:
```suggestion
            ?.filter((affiliation) => affiliation.id)
            .map((affiliation) => distinctUnits.indexOf(affiliation.id) + 1)
```
Men da blir det klaging på at `affiliation.id` kan være null i map :/